api/app/__init__.py

```diff
@@ -146,6 +155,8 @@ def register_blueprint(application):
     application.register_blueprint(status_blueprint)
 
     # delivery receipts
+    ses_callback_blueprint.before_request(requires_no_auth)
```

Comment:
Unless I missed it, we should verify messages sent to this endpoint are from AWS to protect ourselves from DOS attacks. It's unlikely to happen now when we're still early stages but when use picks up we should consider implementing it.

https://docs.aws.amazon.com/sns/latest/dg/SendMessageToHttp.verify.signature.html

Reply:
You're right. This link will give us some existing Python implementations as it doesn't look like there's any official SDK support: https://github.com/boto/boto3/issues/1469